### DISCUSSION TITLE
Autocomplete and using TAB key for selection

### DISCUSSION THREAD

**Discussion by pazziman** [primefaces/community#1001] [PrimeNG] [answered]
Hey all,
as a noob I would really appreciate your help. In the latest version of Autocomplete Component there is the following info: 
![image](https://github.com/primefaces/community/assets/157469228/95e3af6e-63ca-46ec-9f2e-7104b8b9759c)

After updating to the new version, this keyboard feature seems not to work.. I am always forced to press the TAB key twice to move to the next element that can be focused. (The 1st press just selects the item and the cursor stays inside the element).

Do I need to change anything in the component settings? 

Thank you in advance for your help..

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for reporting the issue, we'll address it in the next release. It should work as it is in https://primevue.org/autocomplete/#basic

Issue link: https://github.com/primefaces/primeng/issues/14641
